tools/ingest_new_runs.py

```
@@ -0,0 +1,464 @@
+        if not (parts[0].isdigit() and len(parts[0]) == 8):
+            continue
+
+        try:
+            dt = datetime.strptime(parts[0], "%Y%m%d")
+            iso_week = dt.isocalendar().week
+        except ValueError:
+            continue
+
+        try:
+            df = pd.read_csv(csv_path)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)